README.md

```diff
@@ -6,96 +6,94 @@
 [![GitHub Workflow Status](https://img.shields.io/github/workflow/status/CanalTP/transit_model/Continuous%20Integration?logo=github&style=flat-square)](https://github.com/CanalTP/transit_model/actions?query=workflow%3A%22Continuous+Integration%22)
 [![License: AGPL v3.0](https://img.shields.io/github/license/CanalTP/transit_model?color=9873b9&style=flat-square)](./LICENSE)
 
-`transit_model` is a Rust crate managing transit data by implementing the NTFS
-model (used  in [navitia](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md)). See the
-section [NTFS: Level of Support](#ntfs-level-of-support) for more details about the
-level of support of the NTFS standard.
-
-This repository also provides :
-- (incomplete) [GTFS](http://gtfs.org/) to [NTFS](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) and (soon) NTFS to GTFS conversion.
-- (incomplete) Generation of transfers.
-.
-- Merge [NTFS] (https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md).
-
-## Compile
-
-```bash
-cargo build --release
+**`transit_model`** is a Rust crate to manage, convert and enrich transit
+data.<br>
+This is done by implementing the [NTFS] model (used in [navitia]).
+
+This repository regroups crates that offer enabler-libraries and binaries to
+convert and enrich transit data.
+
+Please check documentation attached to each crate:
+* [**gtfs2netexfr**](gtfs2netexfr/README.md) converts [GTFS] data format into
+  [NeTEx]-France data format.
+* [**gtfs2ntfs**](gtfs2ntfs/README.md) converts [GTFS] data format into [NTFS]
+  data format.
+* [**ntfs2gtfs**](ntfs2gtfs/README.md) converts [NTFS] data format into [GTFS]
+  data format.
+* [**ntfs2netexfr**](ntfs2netexfr/README.md) converts [NTFS] data format into
+  [NeTEx]-France data format.
+* [**ntfs2ntfs**](ntfs2ntfs/README.md) checks and cleans a [NTFS] dataset.
+* [**restrict-validity-period**](restrict-validity-period/README.md) restricts
+  the validity period of a [NTFS] dataset and purges out-of-date data.
+
+## [PROJ] dependency
+
+Based on [PROJ], the [`proj` crate] allows the transformation of
+localization coordinates.
+
+Some `transit_model`'s crates (see each documentation) use [PROJ].<br>
+So it must be installed on the system to compile and use those crates.
+
+### [PROJ] for binaries
+
+Using the [`proj` crate] requires some system-dependencies installation.
+
+The version `6.3` of [PROJ] is needed.
```

Comment:
example 
_Rust requires a version 6+.
Debian based distributions (even the latest Ubuntu), the distributed version is 5. Therefore this library needs to be compiled._

Reply:
Ah, I did some change on this already, but this is more concise :smile: 
I don't know what version precisely are OK: `6 =< v < 7` ?
As for the system version, I figured the user will check her version by any mean she knows, as compiling is always long...

Let you check and maybe discuss it directly?

Reply:
If we don't know about which versions of Proj are compatible, I suggest to mention we're testing our product with `6.3.0` and let the users take their own responsibilities of using anything else than that version.

Reply:
@woshilapin yes it's simpler like that :+1:

Reply:
Changed here: https://github.com/CanalTP/transit_model/pull/655/commits/6c91f23f0149bc957b33a67d617efaaef45c5b9f#diff-04c6e90faac2675aa89e2176d2eec7d8L41-R69